colab_train.ipynb

```diff
@@ -8,8 +8,13 @@
       "collapsed_sections": []
     },
     "kernelspec": {
-      "display_name": "Python 3",
-      "name": "python3"
+      "name": "python3",
+      "display_name": "Python 3.8.2 64-bit",
```

Comment:
we might want to revert this change since we have been using python 3.7 in colab

Reply:
Reverted. Closing this comment now